content/en/blog/_posts/2024-03-15-userns-beta/index.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: blog
+title: "Kubernetes 1.30: User-namespace hits Beta!"
+date: 2024-03-15
+slug: userns-beta
+---
+
+**Authors:** Rodrigo Campos Catelin (Microsoft), Giuseppe Scrivano (Red Hat), Sascha Grunert (Red Hat)
+
+Linux provides different namespaces to isolate processes from each other. A
+typical Kubernetes pod runs within a network namespace to isolate the network
+identity, a PID namespace to isolate the processes, etc.
```

Comment:
For example, a typical and can drop the etc. at the end.

Reply:
Fixed, thanks!